pcdsdevices/slits.py

```diff
@@ -304,3 +307,40 @@ def _aperture_changed(self, *args, **kwargs):
         kwargs.pop('obj',      None)
         # Run subscriptions
         self._run_subs(sub_type=self.SUB_STATE, obj=self, **kwargs)
+
+
+def _rtd_fields(cls, attr_base, range_, **kwargs):
+    padding = max(range_)//10 + 2
+    defn = OrderedDict()
+    for i in range_:
+        attr = '{attr}{i}'.format(attr=attr_base, i=i)
+        suffix = ':RTD:{i}'.format(i=str(i).zfill(padding))
+        defn[attr] = (cls, suffix, kwargs)
+    return defn
+
+
+class RTD(Device, BaseInterface):
```

Comment:
Is there a functional difference between this `RTD` class and the `Thermocouple` class in `sensors.py`?